src/tblib/__init__.py

```diff
@@ -88,3 +97,47 @@ def __tproxy_handler(self, operation, *args, **kwargs):
                 return getattr(self, args[0])
         else:
             return getattr(self, operation)(*args, **kwargs)
+
+    def to_dict(self):
+        """Convert a Traceback into a dictionary representation"""
+        if self.tb_next is None:
+            tb_next = None
+        else:
+            tb_next = self.tb_next.to_dict()
+
+        code = {
+            k: v
+            for k, v in self.tb_frame.f_code.__dict__.items()
+            if k.startswith('co_')
+        }
+        code['co_lnotab'] = code['co_lnotab'].decode('latin1')
+        frame = {
+            'f_globals': self.tb_frame.f_globals,
+            'f_code': code
+        }
+        return {
+            'tb_frame': frame,
+            'tb_lineno': self.tb_lineno,
+            'tb_next': tb_next
+        }
+
+    @classmethod
+    def from_dict(cls, dct):
+        if dct['tb_next']:
+            tb_next = cls.from_dict(dct['tb_next'])
+        else:
+            tb_next = None
+
+        frame = _AttrDict((
+            ('f_globals', dct['tb_frame']['f_globals']),
+            ('f_code', _AttrDict((k, v) for k, v in dct['tb_frame']['f_code'].items()))
+        ))
+        frame['f_code']['co_lnotab'] = frame['f_code']['co_lnotab'].encode('latin1')
```

Comment:
Should this be encoded to something else? Like `hex` encoding. I doubt `latin1` can be used for binary data.

Also, we might not need co_lnotab at all (it just maps bytecode offsets to source linenumbers).